apps/client/src/pages/my/page/confeti/confeti-detail.tsx

```diff
@@ -1,13 +1,23 @@
 import { FestivalCard, Header } from '@confeti/design-system';
-import { PERFORMANCE_DATA } from '@shared/mocks/performance-data';
 import * as styles from './confeti-detail.css';
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { PERFORMANCE_QUERY_OPTIONS } from '@shared/apis/my-confeti';
+import { Performance } from '@shared/types/performance-response';
+import { PERFORMANCE_DATA } from '@shared/mocks/performance-data';
 
 const ConfetiDetail = () => {
+  const { data } = useSuspenseQuery(PERFORMANCE_QUERY_OPTIONS.ALL());
+
+  const allPerformances = [
+    ...data.performances,
+    ...PERFORMANCE_DATA.data.performances.slice(3),
+  ];
+
   return (
     <>
       <Header variant="detail" title="My Confeti" />
       <div className={styles.container}>
-        {PERFORMANCE_DATA.data.performances.map((performance) => (
+        {allPerformances.map((performance: Performance) => (
```

Comment:
```suggestion
        {allPerformances.map((performance) => (
```
FestivalCard 내에서 props의 타입추론이 이루어지고 있어서 map 메서드에서 타입 지정은 불필요해보여요